PubnativeLite/PubnativeLite/VAST/PNLiteVASTPlayerViewController.m

```diff
@@ -600,6 +613,7 @@ - (void)moviePlayBackDidFinish:(NSNotification*)notification {
     [self.playerItem seekToTime:kCMTimeZero];
     [self setState:PNLiteVASTPlayerState_READY];
     [self invokeDidComplete];
+    self.btnClose.hidden = NO;
```

Comment:
I believe this skip feature is only available for Interstitial Ads not in MRect. So you need to add a check such as `self.isInterstitial` in order to show to close button only in that format.  Correct @eros902002 ?

Reply:
Agree. Although not sure if any close button will appear in MRECT.